### PR TITLE
NXP-27472: bootstrap nuxeo-management-rest-api plugin

### PR DESCRIPTION
**Pull request by troger**

### CODE REVIEW

**Review by kevinleturc — DISMISSED** [review]

File: nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/ManagementModule.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Thomas Roger
+ */
+
+package org.nuxeo.rest.management;
```
Comment (ataillefer) [inline nuxeo-management-rest-api/src/main/java/org/nuxeo/rest/management/ManagementModule.java:20]: `org.nuxeo.rest.api.management` maybe?

File: nuxeo-management-rest-api/src/main/resources/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 1
+Bundle-Name: Nuxeo Management REST API
+Bundle-SymbolicName: org.nuxeo.rest.management;singleton:=true
```
Comment (ataillefer) [inline nuxeo-management-rest-api/src/main/resources/META-INF/MANIFEST.MF:4]: `org.nuxeo.rest.api.management`?

File: nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/ManagementFeature.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import org.nuxeo.ecm.core.test.CoreFeature;
+import org.nuxeo.ecm.webengine.test.WebEngineFeature;
+import org.nuxeo.runtime.test.runner.Deploy;
+import org.nuxeo.runtime.test.runner.Features;
+import org.nuxeo.runtime.test.runner.RunnerFeature;
+
+/**
+ * @since 11.1
+ */
+@Features({ CoreFeature.class, WebEngineFeature.class })
```
Comment (ataillefer) [inline nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/ManagementFeature.java]: `CoreFeature` is technically not needed

File: package/src/main/resources/package.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+<title>Nuxeo Management REST API</title>
+<description>
+  <p>The Management REST API adds a set of REST endpoints allowing to manage the Nuxeo Platform.</p>
+</description>
+<home-page>https://doc.nuxeo.com/nxdoc/nuxeo-csv</home-page>
+<vendor>Nuxeo</vendor>
+<installer restart="true" />
+<uninstaller restart="true" />
+<!-- <hotreload-support>false</hotreload-support> -->
+<!-- <require-terms-and-conditions-acceptance>false</require-terms-and-conditions-acceptance> -->
```
Comment (ataillefer) [inline package/src/main/resources/package.xml]: Can remove if not needed

**Review by ataillefer — DISMISSED** [review]

**Review by ataillefer — APPROVED** [review]

**Review by kevinleturc — APPROVED** [review]

**Pull request merged**